Sources/GroundState/variational_montecarlo.hpp

```diff
@@ -154,8 +158,10 @@ class VariationalMonteCarlo {
    */
   void ComputeObservables() {
     for (std::size_t i = 0; i < obs_.size(); ++i) {
-      auto ex = vmc::Expectation(vmc_data_, psi_, *obs_[i]);
-      observable_stats_[obsnames_[i]] = ex;
+      auto local_values = LocalValues(mc_data_.samples, mc_data_.log_values,
+                                      psi_, *obs_[i], sampler_.BatchSize());
+      auto stats = Statistics(local_values, sampler_.BatchSize());
```

Comment:
why do we need to pass explicitly the batchsize to Statistics?

Reply:
Yes, because `Statistics` need to know how many Markov Chains there are.

Reply:
but mc_data_.samples and mc_data.log_values shouldn't already contain this information?

Reply:
No, they don't, but I can add `num_chains` member to `MCResult`.